net-p2p/syncthing/syncthing-0.14.0-r1.ebuild

```diff
@@ -86,14 +86,11 @@ src_install() {
 	if use tools ; then
 		# openrc and systemd service files
 		systemd_dounit "${S}"/src/${EGO_PN}/cmd/strelaysrv/etc/linux-systemd/strelaysrv.service
-		newconfd "${FILESDIR}/${PN}-relaysrv.confd" ${PN}-relaysrv
-		newinitd "${FILESDIR}/${PN}-relaysrv.initd" ${PN}-relaysrv
-
-		keepdir /var/lib/${PN}-relaysrv
```

Comment:
I figured out I was wrong removing those lines. That is, they were incorrect, but  `enewuser` does not create the home directory for the new user by default.